<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>311</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>控制单元</string>
   </property>
   <widget class="QLabel" name="label_motor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">:/images/motor.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_motor_on">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/on.png</normaloff>:/images/on.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_motor_off">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/off.png</normaloff>:/images/off.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_led">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">:/images/led.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_buzzer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">:/images/buzzer.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_matrix">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">:/images/matrix.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_led_off">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/off.png</normaloff>:/images/off.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_buzzer_off">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/off.png</normaloff>:/images/off.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_matrix_off">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/off.png</normaloff>:/images/off.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_led_on">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/on.png</normaloff>:/images/on.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_buzzer_on">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/on.png</normaloff>:/images/on.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_matrix_on">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>105</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="icon">
     <iconset resource="../../gitworkspace/magic/magicprogram/UP-Magic_Demo实训案例/resource/UP-Magic/images.qrc">
      <normaloff>:/images/on.png</normaloff>:/images/on.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>271</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>状态单元</string>
   </property>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Stemp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_humi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>118</red>
          <green>116</green>
          <blue>108</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="NoBrush">
         <color alpha="128">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>湿度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Shumi">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_irda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>红外:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sirda">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_smog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>烟雾:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ssmog">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_miph">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>麦克:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Smiph">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6组二次开发</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
